### DISCUSSION TITLE
How to setup Tasmota so as SONOFF ZBBridge is using one MQTT topic per conected sensor??

### DISCUSSION THREAD

**Discussion by nmaou** [arendst/Tasmota#11999] [Q&A] [answered]
I created a small network with an SONOFF ZBBridge MQTT coordinator, 2 SONOFF zigbee Door sensors and an SONOFF zigbee button. I want ot control the network using Openhab. I am using Mosquitto as an MQTT bridge to Openhab.

I am using **MQTT explorer** to see the messages sent to Mosquitto by the Zbbridge. I noticed that the Zbbridge uses one topic for all the sensors. The topic is "tele/NMAuto/SENSOR" (where NMAuto is the value of the MQTT TOPIC, I defined in the Tasmota MQTT configuration.

Is there a way to setup Tasmota so it uses one MQTT topic per connected sensor?

**Comment by sfromis** [comment, marked as answer]
Not as an option. You can however use [rules](https://tasmota.github.io/docs/Rules/) to catch the codes, and publish your own MQTT message.

**Reply by nmaou** [reply]
Thanks for the info. I will try it.

**Reply by sfromis** [reply]
Oops, I was thinking of RF when I answered. For Zigbee there is an option for
https://tasmota.github.io/docs/Zigbee/#sending-sensor-values-to-separated-mqtt-topics

**Reply by nmaou** [reply]
Thanks

**Reply by nmaou** [reply]
Thanks for the help. I tried it and it worked like a charm.
